tools/src/test/java/org/apache/kafka/tools/consumer/group/ConsumerGroupCommandTestUtils.java

```diff
@@ -92,6 +94,7 @@ static void generator(ClusterGenerator clusterGenerator) {
     static <T> AutoCloseable buildConsumers(int numberOfConsumers,
```

Comment:
Could we have two `buildConsumers` to deal with "assign"/"subscribe" individually?

Reply:
Thank you. I update to define two `buildConsumers`. May you help me take a look? If we're good with current design. I will update to https://github.com/apache/kafka/pull/15908.